CAP/gap/OppositeCategory.gi

```diff
@@ -338,6 +374,10 @@ BindGlobal( "CAP_INTERNAL_INSTALL_OPPOSITE_ADDS_FROM_CATEGORY",
         
     od;
     
+    if HasRangeCategoryOfHomomorphismStructure( opposite_category ) then
+        Assert( 0, [ ] = CheckConstructivenessOfCategory( opposite_category, "IsEquippedWithHomomorphismStructure" ) );
```

Comment:
If the underlying category has itself a partially constructive hom structure, the opposite category will too, so this assertion won't hold true.

Reply:
Done.